chart/templates/statefulset.yaml

```diff
@@ -92,8 +92,7 @@ spec:
 {{- include "vcluster.plugins.volumes" . | indent 8 }}
         - name: helm-cache
           emptyDir: {}
-        - name: binaries
-          emptyDir: {}
+{{ toYaml .Values.controlPlane.statefulSet.persistence.binariesVolume | indent 8 }}
```

Comment:
I'm not entirely sure about this, especially as we do it like this for other values as well above. But what would happen if someone uses the vcluster.yaml from e.g. a previous beta release where `binariesVolume` does not exist in the vcluster.yaml?!

Reply:
It should use the ones defined in the current values.yaml file by default, if it wasn't specified previously. I will validate this too, just to be sure.